core/src/autogluon/core/searcher/bayesopt/utils/comparison_gpy.py

```diff
@@ -1,14 +1,16 @@
-from typing import Optional
+from typing import Optional, List
```

Comment:
Since this seems like a test, should this not live in tests?

Reply:
It is supporting different tests, and may also support examples.